index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
         </section>
       </main>
     </div>
+
+    <div>
+      <section>
+        <div class="garden">
+
+          <div class="water1">
+          <div class="portfolio"><img src="./img/Snapshoot-Portfolio.png" alt="portfolio"></div>
+          <h2 class="tonic">Tonic</h2>
+          <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+          </ul>
+          <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div>
+              <ul class="buttons">
+                  <li><button class="int-button">html</button></li>
```

Comment:
- [ ] Please try to stick to Figma design as much as possible to make your project looks more professional, so try to add some right padding or margins around your project cards to make their end edges visible.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185733927-a13bdc4c-026c-445d-b9d4-35b474b62666.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/185733947-37f63f57-1ac9-4157-a249-d6f90c3072b9.png)